crates/rune/src/compile/options.rs

```diff
@@ -27,6 +39,10 @@ pub struct Options {
     pub cfg_test: bool,
     /// Use the second version of the compiler in parallel.
     pub v2: bool,
+
+    /// Allow use of META_FIELD_GET and META_FIELD_SET
+    #[cfg(feature = "dynamic_fields")]
+    pub dynamic_fields: bool,
```

Comment:
`pub(crate)` since it has a mutator method (else we can set it directly).

The mutator method can be feature gated. This field does not need to be.

Reply:
Yeah I was thinking that but I wasn't 100%.